external/fv3fit/tests/test_jacobian.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import tensorflow as tf
+import pytest
+
+
+from fv3fit._shared.jacobian import (
+    get_jacobians,
+    standardize_jacobians,
+    standardize_jacobians_by_vertical_level,
+)
+
+
+def test_jacobians():
+    n = 5
+    sample = {
+        "a": tf.random.normal((10000, n)),
+        "b": tf.random.normal((10000, n)),
+    }
+
+    sample["field"] = sample["a"]
+
+    profiles = {
+        name: tf.reduce_mean(sample[name], axis=0, keepdims=True) for name in ["a", "b"]
+    }
+
+    def model(x):
+        # no effect from 'b'
+        return {"field": x["a"] + x["b"] * 0}
+
+    jacobians = get_jacobians(model, profiles)
+    normalized = standardize_jacobians(jacobians, sample)
+
+    assert set(normalized) == {"field"}
+    assert set(normalized["field"]) == {"a", "b"}
+
+    np.testing.assert_array_almost_equal(normalized["field"]["a"], np.eye(n))
+    np.testing.assert_array_almost_equal(normalized["field"]["b"], np.zeros((n, n)))
+
+
+@pytest.mark.parametrize(
```

Comment:
This is the only new test, everything above was relocated out of `tests.emulation.test_jacobian.py`